src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -302,37 +294,12 @@ def launch_fluent(
     The allocated machines and core counts are queried from the scheduler environment and
     passed to Fluent.
     """
-    if version == "2d" and gpu:
-        raise GPUSolverSupportError()
-    if show_gui is not None:
-        warnings.warn(
-            "'show_gui' is deprecated, use 'ui_mode' instead",
-            PyFluentDeprecationWarning,
-        )
-    if show_gui or os.getenv("PYFLUENT_SHOW_SERVER_GUI") == "1":
-        ui_mode = UIMode.GUI
-    del show_gui
-    if ui_mode is None:
-        # Not using NO_GUI in windows as it opens a new cmd or
-        # shows Fluent output in the current cmd if start <launch_string> is not used
-        ui_mode = UIMode.HIDDEN_GUI if is_windows() else UIMode.NO_GUI
-    if isinstance(ui_mode, str):
-        ui_mode = UIMode(ui_mode)
-    if graphics_driver is None:
-        graphics_driver = "auto"
-    graphics_driver = str(graphics_driver)
-    graphics_driver = (
-        FluentWindowsGraphicsDriver(graphics_driver)
-        if is_windows()
-        else FluentLinuxGraphicsDriver(graphics_driver)
-    )
     fluent_launch_mode = _get_fluent_launch_mode(
         start_container=start_container,
         container_dict=container_dict,
         scheduler_options=scheduler_options,
     )
     del start_container
-    mode = _get_mode(mode)
     argvals = locals().copy()
     _process_invalid_args(dry_run, fluent_launch_mode, argvals)
```

Comment:
We have already removed invalid args for `PIM` and `DockerContainer` launcher from constructor signature therefore `_process_invalid_args` is not required in these 2 launchers.

The invalid args are supported for `Standalone` launcher only therefore `if fluent_launch_mode != enums.LaunchMode.STANDALONE:` is not valid inside `Standalone` launcher.

The first condition check `if dry_run and fluent_launch_mode != enums.LaunchMode.CONTAINER:` is only possible in `launch_fluent` because `dry_run` is not supported for `Standalone` and `PIM` launcher.

Therefore I think `_process_invalid_args` should be as it is here.

Reply:
Is it possbile to replace `_process_invalid_args` with a decorated `__init__` function in the individual launcher class? Some usage like this:
```
class PIMLauncher:
    @warn_unknown_keywards
    def __init__(...):  # this won't have `env` in the keyward list
        ...
        
launcher = PIMLauncher(..., env=...)
# returns a PIM launcher but will print a warning: Unknown keyword arguments {'env'} are not processed.
```
Make sure the `__init__` docstring is not changed by wrapping a decorator.

Reply:
We will try this. Thank you.

Reply:
We have to introduce **kwargs in `def __init__(...):` then only it is possible to use `PIMLauncher(..., env=...)`, but we have already removed kwargs from all constructors and we don't want to use it.

Reply:
We have found another solution for this. We have moved `_process_invalid_args` to `create_launcher` where all required arguments are available i.e. `dry_run`, `fluent_launch_mode`, `kwargs` for `_process_invalid_args`. 

Standalone - 

![image](https://github.com/ansys/pyfluent/assets/106588300/81c11141-7e7f-4153-8a90-c8467e6b14d7)

Container 

![image](https://github.com/ansys/pyfluent/assets/106588300/367c1a57-310b-47d8-a1ac-51c5ce5a5639)